appium/options/android/espresso/activity_options_option.py

```diff
@@ -19,7 +19,7 @@
 
 from appium.options.common.supports_capabilities import SupportsCapabilities
```

Comment:
should below option also be refactored?

Reply:
I have refactored 
`ActivityOptionsOption`
`AppLocaleOption`
`IntentOptionsOption`
`ShowGradleLogOption`

Reply:
I assume it should also be moved to a common module like other options which were previously in separate modules